phpBB/styles/prosilver/template/pm.js

```diff
@@ -6,6 +6,10 @@ var to_modal = document.getElementById('pm-to-model');
 var to_btn = document.getElementById("pm-to-button");
 var to_span = document.getElementsByClassName("pm-to-button-close")[0];
 
+var bb_modal = document.getElementById('pm-bbcode-model');
```

Comment:
You should be using jquery for the js

Reply:
I see, will something like this do?
https://codepen.io/pairdocs/pen/JOpWPz

Reply:
There are [phpbb.alert()](https://github.com/phpbb/phpbb/blob/release-3.2.2/phpBB/assets/javascript/core.js#L92) and [phpbb.confirm()](https://github.com/phpbb/phpbb/blob/release-3.2.2/phpBB/assets/javascript/core.js#L179) functions in core.js, maybe you can use one of those?

Reply:
Looks like this what I need. Is there any sample code I can look at?

Reply:
Examples are [here for alert()](https://github.com/phpbb/phpbb/blob/release-3.2.2/phpBB/assets/javascript/core.js#L288) and [here for confirm()](https://github.com/phpbb/phpbb/blob/release-3.2.2/phpBB/assets/javascript/core.js#L349-L363).